<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sysinfo</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="generaltab">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Current Language:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2" colspan="2">
        <widget class="QLineEdit" name="curLanguageLineEdit"/>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Available Languages:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2" colspan="2">
        <widget class="QComboBox" name="languagesComboBox"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Version</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="2">
        <widget class="QComboBox" name="versionComboBox">
         <item>
          <property name="text">
           <string>Os</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>QtCore</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>WrtCore</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Webkit</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ServiceFramework</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>WrtExtensions</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Service Provider</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Netscape Plugin</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Webapp</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Firmware</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="3" colspan="2">
        <widget class="QLineEdit" name="versionLineEdit"/>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Feature supported</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2" colspan="2">
        <widget class="QComboBox" name="featureComboBox">
         <item>
          <property name="text">
           <string>Bluetooth</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Camera</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>FmRadio</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Irda</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Led</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Memcard</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Usb</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Vibrate</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>WLan</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Sim</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Location</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>VideoOut</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Haptics</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="4">
        <widget class="QLineEdit" name="featuresLineEdit"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="sysdevicetab">
      <attribute name="title">
       <string>Device</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <widget class="QProgressBar" name="batteryLevelBar">
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item row="0" column="2" rowspan="4" colspan="2">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Power state</string>
         </property>
         <widget class="QRadioButton" name="radioButton_3">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>100</y>
            <width>101</width>
            <height>24</height>
           </rect>
          </property>
          <property name="text">
           <string>Wall Power</string>
          </property>
         </widget>
         <widget class="QRadioButton" name="radioButton">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>20</y>
            <width>151</width>
            <height>24</height>
           </rect>
          </property>
          <property name="text">
           <string>Unknown Power</string>
          </property>
         </widget>
         <widget class="QRadioButton" name="radioButton_2">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>60</y>
            <width>121</width>
            <height>24</height>
           </rect>
          </property>
          <property name="text">
           <string>Battery Power</string>
          </property>
         </widget>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="batteryChargingCheckBox">
         <property name="text">
          <string>Battery Charging</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>IMEI:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="ImeiLabel">
         <property name="text">
          <string>xx</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_19">
         <property name="text">
          <string>IMSI:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="imsiLabel">
         <property name="text">
          <string>xx</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_20">
         <property name="text">
          <string>Manufacturer:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="manufacturerLabel">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_21">
         <property name="text">
          <string>Model</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLabel" name="modelLabel">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_22">
         <property name="text">
          <string>Product:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLabel" name="productLabel">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QCheckBox" name="deviceLockCheckBox">
         <property name="text">
          <string>Device locked</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_23">
         <property name="text">
          <string>Sim status:</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1" colspan="2">
        <widget class="QComboBox" name="simComboBox">
         <item>
          <property name="text">
           <string>Sim not available</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Single available</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Dual available</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Locked</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="8" column="3">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>201</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="label_24">
         <property name="text">
          <string>current Profile:</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1" colspan="2">
        <widget class="QComboBox" name="profileComboBox">
         <item>
          <property name="text">
           <string>Unknown Profile</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Silent Profile</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Normal Profile</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Loud profile</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Vib profile</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Offline profile</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Powersave profile</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Custom profile</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="label_25">
         <property name="text">
          <string>Input method:</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QComboBox" name="inputMethodComboBox">
         <item>
          <property name="text">
           <string>Keys</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Keypad</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Keyboard</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Single Touch</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Multi Touch</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Mouse</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="sysdisplaytab">
      <attribute name="title">
       <string>Display</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>brightness</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="brightnessLineEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>color depth</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="colorDepthLineEdit"/>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QCheckBox" name="screenLockCheckBox">
         <property name="text">
          <string>screen lock on?</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="sysmemorytab">
      <attribute name="title">
       <string>Memory</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QComboBox" name="diskComboBox">
         <item>
          <property name="text">
           <string>Total Disk Space</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Available Disk Space</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Type of Volume</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>list of volumes</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QComboBox" name="volumesComboBox"/>
       </item>
       <item row="2" column="1" colspan="2">
        <widget class="QLineEdit" name="diskSpaceLineEdit"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="sysnetworktab">
      <attribute name="title">
       <string>Network</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Cell Network Status</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4" colspan="2">
        <widget class="QComboBox" name="comboBox_4">
         <item>
          <property name="text">
           <string>Undefined</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>No Network</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Emergency Only</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Searching</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Busy</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Home network</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Denied</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Roaming</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Network Signal Strength</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4" colspan="2">
        <widget class="QComboBox" name="comboBox_5">
         <item>
          <property name="text">
           <string>Unknown</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Gsm</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Cdma</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Wcdma</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Wlan</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Ethernet</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="6" colspan="2">
        <widget class="QLineEdit" name="signalLineEdit"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Cell Id:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="4">
        <widget class="QLineEdit" name="cellIdLineEdit"/>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Location Area Code</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3" colspan="2">
        <widget class="QLineEdit" name="lacLineEdit"/>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>current MMC</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2" colspan="3">
        <widget class="QLineEdit" name="curMmcLineEdit"/>
       </item>
       <item row="4" column="5" colspan="2">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>home MMC</string>
         </property>
        </widget>
       </item>
       <item row="4" column="7">
        <widget class="QLineEdit" name="homeMmcLineEdit"/>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>current MNC</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2" colspan="3">
        <widget class="QLineEdit" name="curMncLineEdit"/>
       </item>
       <item row="5" column="5" colspan="2">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>home MNC</string>
         </property>
        </widget>
       </item>
       <item row="5" column="7">
        <widget class="QLineEdit" name="homeMncLineEdit"/>
       </item>
       <item row="6" column="0" colspan="4">
        <widget class="QCheckBox" name="checkBox">
         <property name="text">
          <string>Wlan Accessable</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="2">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>operator Name</string>
         </property>
        </widget>
       </item>
       <item row="7" column="3" colspan="2">
        <widget class="QLineEdit" name="operatorNameLineEdit"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
